1-callbacks/solution.js

```diff
@@ -31,17 +31,35 @@ node solution.js name1 name2 name3
 5. another challenge is: after you solve the challenge using callback style, in another file promisify the callback and solve it again
 ** give a look to node.js util.promisify, avoid to alter the validate-user.file **
 */
+const validateUser = require('./validate-user');
+const { argv } = require('node:process');
 
 function solution() {
-    // YOUR SOLUTION GOES HERE
-
-    // you get your 5 names here
-
-    // iterate the names array and validate them with the method
-
-    // log the final result
+  // you get your 5 names here
+  const defaultNames = ['Melani', 'John', 'Mary', 'Fabio', 'Leidy'];
+  // If there were names in the terminal access to them and asigned to names variable
+  const names = argv.slice(2).length > 0 ? argv.slice(2) : defaultNames;
+
+  let validUsers = 'Success\n',
+    invalidUsers = '\nFailure\n\n',
+    counter = 0;
+
```

Comment:
As a good practice for readability, you can declare the variables separately, with its own `let` statement